drivers/filesystems/reiserfs/src/read.c

```diff
@@ -605,7 +605,7 @@ RfsdReadFile(IN PRFSD_IRP_CONTEXT IrpContext)
         Length = IoStackLocation->Parameters.Read.Length;
         ByteOffset = IoStackLocation->Parameters.Read.ByteOffset;
         
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
No variadic arguments ?

Reply:
I'm guessing it's about __FUNCTION__ being a literal (MSVC) vs a static WCHAR buffer (standard C)